generate_from_csv.py

```diff
@@ -151,6 +151,7 @@ def generate_from_csv(args):
     # save path and options for CSV timeseries
     # all paths are relative to output file
     target_path = path.dirname(args.output)
+
```

Comment:
Similar argumentation as for generate.py, this is the only change in this file.

Reply:
so you like the line here or not? When changing back the target_path I liked a blank line here. But I guess it does not really matter much

Reply:
No, I don't like the newline. It disrupts the initialization block and is the only change to this file, making the diff bigger as it needs to be.